### ISSUE TITLE
How can I pass a controller?

### ISSUE THREAD

**Issue by IngwiePhoenix** [baseprime/grapnel#53]
So I am currently building a new frontend structure for my app, and I wanted to use Grapnel for that. The structure involves using controllers. Actually, the app is driven by Laravel 5.1, but some parts are actually simpler via Webpack and JSX. So Laravel simply serves the page template that I want to drop my Controller's rendered output into.

My HTML is something like this (simplified):

``` html
<div class="docs sidebar" id="DocBar">
</div>
<div class="docs body" id="DocBody">
</div>
```

Now I want to do something like this with Grapnel:

``` javascript
app.get("/docs", function(req, e, next){
    require.ensure([
        "BIRD3/App/Controllers/DocsController"
    ], function(require){
        var Controller = require("BIRD3/App/Controllers/DocsController");
        next(new Controller); // <----
    }, "DocsController");
});
```

Basically, I want to _keep going_ untill the PJAX data is loaded and then run the `.render()` method in my class.

Is there a way I can use `next()` like I did above, or do I need to use events to make this work?
